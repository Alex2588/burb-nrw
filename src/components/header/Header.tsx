import React from 'react';
import { Link } from 'react-router-dom';

import './header.styles.scss';

const Header = () => (
  <div className='header'>
    <Link className='logo-container' to='/'>
      BURB-NRW
    </Link>
    <div className='options'>
      <Link className='option' to='/shows'>
        SHOWS
      </Link>
    </div>
  </div>
);

export default Header;