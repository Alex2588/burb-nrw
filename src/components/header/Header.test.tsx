import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import { render, screen } from '@testing-library/react';

import Header from './Header';

test('renders show-item component', async () => {
    const { container} = render(<Header />, { wrapper: MemoryRouter });

    expect(screen.getByText('SHOWS')).toHaveAttribute('href', '/shows');
    expect(container.firstChild).toMatchSnapshot();
});