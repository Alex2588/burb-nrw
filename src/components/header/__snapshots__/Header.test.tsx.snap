// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders show-item component 1`] = `
<div
  class="header"
>
  <a
    class="logo-container"
    href="/"
  >
    BURB-NRW
  </a>
  <div
    class="options"
  >
    <a
      class="option"
      href="/shows"
    >
      SHOWS
    </a>
  </div>
</div>
`;
